### PR TITLE
[Bugfix:Forum] Disable scroll and drag on popup

### PR DESCRIPTION
**Pull request by teazhi**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant
* [ ] Screenshots are attached to Github PR if visual/UI changes were made

### What is the current behavior?

Previously able to drag popup around when editing threads, as well as scroll in the background behind/around the modal.

### What is the new behavior?
Disabled the draggable feature and disabled the page scroll when modal is open.

### Other information?

Testing:
Edit a thread in discussion forum and attempt to drag the popup around and scroll in the area around the popup.

### CODE REVIEW

**Comment by VVVITH** [comment]
could you please attach some pictures and give more detail info about what are you tring to solve? or attach a relate issue? I don't understand your description sorry.

**Comment by oskurtt** [comment]
Tested branch and everything looks good. One small thing to note, please be consistent with programming convention. For example, when/where to indent, spacing, variable naming and etc.

**Review by DarthNyan — CHANGES_REQUESTED** [review]
Few code quality things I noticed. Otherwise, it works to disable dragging.

**Review by RitaLei123 — APPROVED** [review]
The three step review process:
Is the proposed change a good idea?
Yes, it is unnecessary for the box to be draggable and the background to be scrollable. When the screen size is smaller, the box is harder to use if it’s been dragged.

Did the author take the right approach?
It appears that the author removed the code that allows the box to be draggable.

Are all the implementation details correct?
Yes, the implementation details are correct.

Manual testing:
Reproduce on main: The box was draggable and the background was scrollable.
Check topic branch
Reproduce on main again: The box is no longer draggable and the background is not scrollable.

A few other things to check:
If a UI change: light/dark modes, different screen sizes: It works for different screen sizes.

**Review by RitaLei123 — COMMENTED** [review]
Drag and Scroll (For future development, New Feature was not applied to these areas)
- Calendar
- Office Hours Queue --|> Open New Queue
                                   --|> Edit Existing Queue
                                   --|> Edit Announcement
                                   --|> Send Message
- Course Materials --|> All the Blue Buttons
- Manage Student --|> Edit Student
                              --|> All the Blue Buttons
- Manage Graders --|> Edit Grader
                              --|> Demote Grader
                              --|> All the Blue Buttons
- Student Photos --|> Upload Student Photos

File: site/app/templates/forum/CreatePost.twig [hunk 1]
```diff
@@ -88,7 +88,7 @@
...
                 <a class="post_button delete-post-button key_to_click" tabindex = "0" onClick="deletePostToggle({{ post_buttons.delete.ud_toggle_status }}, {{ post.thread_id }}, {{ post.id }}, '{{ post.author_user_id }}', '{{ post_date }}', '{{ csrf_token }}' )" title="{{ post_buttons.delete.ud_button_title }}" aria-label="{{ post_buttons.delete.ud_button_title }}"><i class="fa {{ post_buttons.delete.ud_button_icon }}"></i></a>
-                <a class="post_button edit-post-button key_to_click" tabindex = "0" onClick="showEditPostForm({{ post.id }}, {{ post.thread_id }}, {{ post_buttons.edit.shouldEditThread }}, {{ render_markdown?1:0 }}, '{{ csrf_token }}')" title="{{ post_buttons.edit.edit_button_title }}" aria-label="{{ post_buttons.edit.edit_button_title }}"><i class="fas fa-edit"></i></a>
+                <a class="post_button edit-post-button key_to_click" tabindex = "0" onClick="showEditPostForm({{ post.id }}, {{ post.thread_id }}, {{ post_buttons.edit.shouldEditThread }}, {{ render_markdown?1:0 }}, '{{ csrf_token }}');" title="{{ post_buttons.edit.edit_button_title }}" aria-label="{{ post_buttons.edit.edit_button_title }}"><i class="fas fa-edit"></i></a>
```
Comment (williamjallen) [inline site/app/templates/forum/CreatePost.twig:91]: This change is unrelated to the rest of the PR and should be reverted.

File: site/app/templates/generic/Popup.twig [hunk 2]
```diff
@@ -1,16 +1,16 @@
 {# This is the control shown/hidden #}
-<div class="popup-form" id="{% block popup_id %}#popup-form{% endblock %}" style="display: none;">
+<div class="popup-form" id="{% block popup_id %}popup-form{% endblock %}" style="display: none;">
     {% block form %}
         {# This lets us center the window #}
-        <div class="popup-box" onclick="{{ block('close_click_action') }}">
+        <div class="popup-box" onClick="{{ block('close_click_action') }}">
```
[the remark this answers is not in the document]
Comment (williamjallen) [reply site/app/templates/generic/Popup.twig:5]: @DarthNyan is correct here.  This change should be reverted.

**Review by williamjallen — CHANGES_REQUESTED** [review]

**Review by VVVITH — APPROVED** [review]
Work as discribed. I have removed the unrelated code changes. Need more reviews and merge.

**Review by DarthNyan — APPROVED** [review]
Looks good to me. You've made a few whitespace changes that I don't think are necessary, but they also don't hurt readability so I think they're fine to stay.

**Comment by VVVITH** [comment]
> Looks good to me. You've made a few whitespace changes that I don't think are necessary, but they also don't hurt readability so I think they're fine to stay.

I think the whitespace changes is ok. The changes is going to match the tabs to make it more clear in my view.

**Review by cjreed121 — APPROVED** [review]

**Pull request merged**